package.json

```diff
@@ -1,50 +1,46 @@
 {
-  "name": "graphql-query-rewriter",
+  "name": "@graphql-query-rewriter/core",
   "version": "0.0.1",
   "description": "",
-  "keywords": [],
-  "main": "dist/index.umd.js",
-  "module": "dist/index.es5.js",
-  "typings": "dist/types/index.d.ts",
+  "keywords": [
+    "graphql",
+    "query",
+    "rewriter",
+    "ast",
+    "schema"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "files": [
     "dist"
   ],
   "author": "David Chanin <chanindav@gmail.com>",
   "repository": {
     "type": "git",
-    "url": "https://github.com/ef-eng/graphql-query-rewriter"
+    "url": "https://github.com/ef-en/graphql-query-rewriter"
   },
   "license": "MIT",
   "engines": {
-    "node": ">=6.0.0"
+    "node": ">=10.0.0"
   },
   "scripts": {
-    "lint": "tslint  --project tsconfig.json -t codeFrame 'src/**/*.ts' 'test/**/*.ts'",
-    "prebuild": "rimraf dist",
-    "build": "tsc --module commonjs && rollup -c rollup.config.ts && typedoc --out docs --target es6 --theme minimal --mode file src",
```

Comment:
I think it's probably fine to get rid of rollup, just out of curiosity why remove it? Is it related to updating the node version? This wall all from https://github.com/alexjoverm/typescript-library-starter

Reply:
Yeah I saw the similarities, I reality I never saw something similar to that setup. If you look around all libs that mainly support node all use similar tsconfig and output commonjs code with types near the js code. So unless we need to do something fancy there is no need at this point to support es5/umd/other types of import.